NitroxClient/MonoBehaviours/Gui/MainMenu/ConsoleJoinServer.cs

```diff
@@ -92,6 +97,25 @@ public void OnConsoleCommand_warpto(NotificationCenter.Notification n)
             }
         }
 
+        public void OnConsoleCommand_sub(NotificationCenter.Notification n)
+        {
+            if (n?.data?.Count > 0)
+            {
+                string subName = (string)n.data[0];
+
+                switch (subName.ToLowerInvariant())
+                {
+                    case "cyclops":
+                        NitroxServiceLocator.LocateService<Vehicles>().CreateVehicle(TechType.Cyclops, Optional<string>.Empty(), MainCamera.camera.transform.position + 20f * MainCamera.camera.transform.forward, Quaternion.LookRotation(MainCamera.camera.transform.right), Optional<List<InteractiveChildObjectIdentifier>>.Empty());
```

Comment:
Shouldn't we generate a GUID here to assign to the cyclops?  Also, I think this might be the place that we want to `BroadcastCreatedVehicle` and not in the create method above.